src/submission/upload/index.jsx

```diff
@@ -21,12 +25,21 @@ export default class Upload extends Component {
 
   componentDidMount() {
     const { code, pollSubmission } = this.props
+    /* TODO
```

Comment:
Just a reminder that we should watch what happens here in certain cases, like a clean file.